src/components/Loader/people/PeoplePage.tsx

```diff
@@ -0,0 +1,97 @@
+/* eslint-disable max-len */
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { getPeople } from '../../../api';
+import { Person } from '../../../types';
+import { Loader } from '../Loader';
+import { PeopleRow } from './PeopleRow';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [isError, setIsError] = useState('');
+
+  useEffect(() => {
+    const fetchData = async () => {
+      setIsLoading(true);
+      try {
+        const peopleArray = await getPeople();
+
```

Comment:
that is not good idea to use Array in our variables, for example, we can use peopleFromServer